src/FiltrationGroupComponent/FiltrationGroupComponent.tsx

```diff
@@ -60,41 +60,39 @@ export class FiltrationGroupComponent extends React.Component<Props, State> {
 				"last"
 			);
 
-			const groupItems = this.getChildElements(groupRef, searchClasses);
-
 			if (firstChildElement) {
 				if (
 					firstChildElement.classList.contains(
 						SearchClasses.KitFiltrationGroup
 					)
 				) {
-					heightLine +=
-						firstChildElement.getBoundingClientRect().height / 2;
+					const firstChildElementHeight = firstChildElement.getBoundingClientRect()
+						.height;
 
-					positionTop =
-						groupItems.length > 1
-							? firstChildElement.getBoundingClientRect().height /
-							  2
-							: 0;
-				}
+					const theLine = firstChildElement.querySelector(
```

Comment:
Я уж по-быстрому называл переменную, чтоб не пересечься где-нибудь. Лучше переименовать нормально, типа
```suggestion
					const labelLine = firstChildElement.querySelector(
```